assets/Icons/raw/lock.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="224" height="280"><path fill-rule="evenodd" d="M20 112h184c11.046 0 20 8.954 20 20v128c0 11.046-8.954 20-20 20H20c-11.046 0-20-8.954-20-20V132c0-11.046 8.954-20 20-20zM112 0c43.078 0 78 32.594 78 72.8V112H34V72.8C34 32.594 68.922 0 112 0zM66 112h92V78.4c0-25.626-20.595-46.4-46-46.4S66 52.774 66 78.4V112z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="224" height="280" viewBox="0 0 224 280"><path fill-rule="evenodd" d="M20 112h184c11.046 0 20 8.954 20 20v128c0 11.046-8.954 20-20 20H20c-11.046 0-20-8.954-20-20V132c0-11.046 8.954-20 20-20zM112 0c43.078 0 78 32.594 78 72.8V112H34V72.8C34 32.594 68.922 0 112 0zM66 112h92V78.4c0-25.626-20.595-46.4-46-46.4S66 52.774 66 78.4V112z"/></svg>
```

Comment:
Is it normal that these three SVGs all have different dimensions?

Reply:
No, we are waiting for updated SVGs that are square and specifically 240x240 I think.